tests/integration/cocoon/test_cocoon_failures.py

```diff
@@ -239,6 +279,7 @@ def test_load_from_data_frame_failure(
         property_columns,
         properties_scope,
         file_type,
+        sub_holding_keys,
         expected_exception,
```

Comment:
Raise a separate JIra to refactor this test to assert on the content of the error as well as the fact that one got raised, i.e. make sure the correct error gets raised

Reply:
Done.